app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class QuotesController < ApplicationController
+  def index
+    @quotes = QuoteFacade.search_term(params[:query])
+    json = QuoteService.search_by_input(params[:query])
+    @total = json[:totalCount]
```

Comment:
I agree with your self assessment that this feels smelly. the `search_term` method in your facade class is calling the same `search_by_input` method in your service so you are hitting the same endpoint twice. I will add some thoughts about a better way to handle this down in the comments for the facade class.